bbb-meeting-api/src/main/scala/org/bigbluebutton/api/signOut/SignOutController.scala

```diff
@@ -0,0 +1,54 @@
+package org.bigbluebutton.api.signOut
+
+import akka.http.scaladsl.model.headers.RawHeader
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.server.Directives.{complete, get, parameter, pathPrefix}
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.server.Directives._
+import com.softwaremill.session.CsrfDirectives.setNewCsrfToken
+import com.softwaremill.session.CsrfOptions.checkHeader
+import com.softwaremill.session.SessionDirectives.{optionalSession, setSession}
+import com.softwaremill.session.SessionOptions.{oneOff, usingCookies}
+import org.bigbluebutton.api.ControllerStandard
+import org.bigbluebutton.model.ApiResponseStandard
+
+case object SignOutController extends ControllerStandard {
+  val route: Route = (pathPrefix("signOut") & extractLog) { log =>
+    log.info("signOut")
+    get {
+      parameter(
+        "sessionToken".as[String],
+        "checksum".as[String],
```

Comment:
This endpoint shouldn't require a checksum since it uses a sessionToken. The current API doesn't need a checkum on this endpoint.